src/plugins/embeddable/public/lib/panel/embeddable_panel_error.tsx

```diff
@@ -62,42 +59,29 @@ export function EmbeddablePanelError({
 
     return () => subscription.unsubscribe();
   }, [editPanelAction, embeddable]);
-  useEffect(() => {
-    if (!ref.current) {
-      return;
-    }
-
-    if (!embeddable.catchError) {
-      const errorEmbeddable = new ErrorEmbeddable(error, { id: embeddable.id });
-      setNode(errorEmbeddable.render());
-
-      return () => errorEmbeddable.destroy();
-    }
-
-    const renderedNode = embeddable.catchError(error, ref.current);
-    if (isFunction(renderedNode)) {
-      return renderedNode;
-    }
-    if (isPromise(renderedNode)) {
-      renderedNode.then(setNode);
-    } else {
-      setNode(renderedNode);
-    }
-  }, [embeddable, error]);
 
   return (
-    <EuiPanel
-      element="div"
-      className="embPanel__content embPanel__content--error"
-      color="transparent"
-      paddingSize="none"
-      data-test-subj="embeddableError"
-      panelRef={ref}
-      role={isEditable ? 'button' : undefined}
-      aria-label={isEditable ? ariaLabel : undefined}
-      onClick={handleErrorClick}
-    >
-      {node}
-    </EuiPanel>
+    <EuiEmptyPrompt
```

Comment:
Would it make sense to also include a `title` prop (along with an appropriate `titleSize`) for this `EuiEmptyPrompt` component (either dynamic or static)? If a static title, would something like `Unable to render visualization` work?

Reply:
I tried to add that similarly to the button, but it requires significant changes in the code to get the embeddable type. I would rather leave that without the title prop. Especially since the error appearance is similar to what we had before.